frontend/components/ModalBox/ModalBox.tsx

```diff
@@ -0,0 +1,20 @@
+export const ModalBox: React.FC = () => {
+    return (
+        <>
+            <link
+                rel="stylesheet"
+                href="https://cdn.jsdelivr.net/npm/bulma@0.9.3/css/bulma.min.css"
+            ></link>
+            <div className="modal">
+                <div className="modal-background"></div>
+                <div className="modal-content">
+                    <p>Dit is een test</p>
+                </div>
+                <button
+                    className="modal-close is-large"
+                    aria-label="close"
+                ></button>
+            </div>
+        </>
+    );
+};
```

Comment:
There is already a Modal Component. And there is no need to import bulma like that. It is installed as a dependency and included in the globals.scss file.

Reply:
Apparently I am blind. I will use the existing modal component!